R/sidebar.R

```diff
@@ -96,14 +96,26 @@ layout_sidebar <- function(
   fillable = FALSE,
   fill = TRUE,
   bg = NULL,
-  border = TRUE,
-  border_radius = TRUE,
+  border = NULL,
+  border_radius = NULL,
   height = NULL
 ) {
   if (!inherits(sidebar, "sidebar")) {
     abort("`sidebar` argument must contain a `bslib::sidebar()` component.")
   }
 
+  if (!(is.null(border) || rlang::is_bare_logical(border, n = 1))) {
+    abort("`border` must be `NULL`, `TRUE`, or `FALSE`.")
+  } else if (!is.null(border) && is.na(border)) {
+    border <- NULL
+  }
```

Comment:
```suggestion
  if (!identical(border, NULL) || !isTRUE(border) || !isFALSE(border)) {
    abort("`border` must be `NULL`, `TRUE`, or `FALSE`.")
  }
```

Reply:
`isFALSE()` was added in 3.5.0, which I believe makes it safe to use here.

Reply:
2883099549f1e97ad5a3f9f7e9ba6f7dd27c29a1